src/Illuminate/Http/Resources/Json/AnonymousResourceCollection.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Illuminate\Http\Resources\Json;
 
+#[\AllowDynamicProperties]
```

Comment:
Fixes `Creation of dynamic property Illuminate\Http\Resources\Json\AnonymousResourceCollection::$preserveKeys is deprecated`